src/main/java/com/google/cloud/tools/managedcloudsdk/install/InstallerFactory.java

```diff
@@ -54,7 +58,12 @@ public InstallerFactory(OsInfo osInfo, boolean usageReporting) {
         getInstallScriptProvider(),
         usageReporting,
         messageListener,
-        new CommandExecutorFactory());
+        new CommandExecutorFactory(),
+        new AsyncStreamConsumer<>(
```

Comment:
I feel like these could probably be simplified, or not even passed in, since the messageListener is already passed into the Installer construction.

Reply:
The message listener isn't necessary here. For example, you could collect and parse the output for use somewhere else.

Reply:
Oh never mind, I see what you're saying.